app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryUpdate.kt

```diff
@@ -24,7 +24,7 @@ class CustomDrugEntryUpdate : Update<CustomDrugEntryModel, CustomDrugEntryEvent,
       is EditFrequencyClicked -> dispatch(ShowEditFrequencyDialog(model.frequency, model.drugFrequencyToFrequencyChoiceItemMap!!.toList().map { it.second }))
       is FrequencyEdited -> next(model.frequencyEdited(event.frequency), SetDrugFrequency(model.drugFrequencyToFrequencyChoiceItemMap!![event.frequency]!!.label))
       is AddMedicineButtonClicked -> createOrUpdatePrescriptionEntry(model, event.patientUuid)
-      is CustomDrugSaved, ExistingDrugRemoved -> dispatch(CloseSheetAndGoToEditMedicineScreen)
+      is CustomDrugSaved, ExistingDrugRemoved -> next(model.drugInfoProgressStateLoaded(), CloseSheetAndGoToEditMedicineScreen)
```

Comment:
I don't think it matters for us to set back the progress state to be done here since we will be closing the sheet. So we can probably drop this commit. wdyt?